### ISSUE TITLE
fix Netperf test module when throuhgput is zero

### ISSUE THREAD

**Issue by jtluka** [jpirko/lnst#179]
When netperf fails with zero throughput, the division by zero error occurs.

```
2016-10-07 16:47:23    (testmachine2)        -   DEBUG: compiled command: netperf -H 192.168.10.1 -f k -P 0 -l 60 -t TCP_STREAM -I 99,5 -i 5,5 -L 192.168.10.2 -T5,5 -- -k "THROUGHPUT, LOCAL_CPU_UTIL, REMOTE_CPU_UTIL, CONFIDENCE_LEVEL, THROUGHPUT_CONFID"
2016-10-07 16:47:23    (testmachine2)        -   DEBUG: running as client...
2016-10-07 16:52:43    (testmachine2)        -   DEBUG: 
    THROUGHPUT=0.00
    LOCAL_CPU_UTIL=-1.00
    REMOTE_CPU_UTIL=-1.00
    CONFIDENCE_LEVEL=99
    THROUGHPUT_CONFID=-nan
    
2016-10-07 16:52:43    (testmachine2)        -   DEBUG: 
    Traceback (most recent call last):
      File "/usr/lib/python2.7/site-packages/lnst/Common/NetTestCommand.py", line 166, in _run
        self._cmd_cls.run()
      File "/var/cache/lnst/194eb6882951000737fbb280b45670ef/Netperf.py", line 580, in run
        (rv, res_data) = self._run_client(cmd)
      File "/var/cache/lnst/194eb6882951000737fbb280b45670ef/Netperf.py", line 507, in _run_client
        percentual_deviation = (rate_deviation / rate) * 100
    ZeroDivisionError: float division by zero
```

**Comment by olichtne** [comment]
Sent a patch for this: https://lists.fedorahosted.org/archives/list/lnst-developers@lists.fedorahosted.org/message/D6BEZEZMZTBI7OZC5DBDDYAKFU47FIA3/

**Comment by olichtne** [comment]
Fixed in 9a1d03186daac2136ae56b446d79f200c8788a46
